src/esportsbot/cogs/AdminCog.py

```diff
@@ -38,6 +38,18 @@ async def clear_messages(self, ctx, amount=5):
             }
         )
 
+    @commands.check(is_dev)
+    @commands.command(name="version", help="Print the bot's version string")
+    async def print_version(self, ctx):
+        try:
+            version_file_path = os.path.join(os.path.dirname(os.path.dirname(__file__)), "version.txt")
+            print(version_file_path)
+            with open(version_file_path, "rt") as version_file:
```

Comment:
Would it not be better to read this in when the cog is loaded?

Reply:
Agreed - late night pull  requests are never a good idea.